### PR TITLE
Bug 1827550: Update the custom node selector example.

### PR DESCRIPTION
**Pull request by timflannagan**

### CODE REVIEW

**Comment by timflannagan1** [comment]
@lamek PTAL

**Comment by timflannagan1** [comment]
Should we also add a note to the header section that specifies that these node labels must exist?

File: metering/configuring_metering/metering-common-config-options.adoc [hunk 1]
```diff
@@ -82,22 +82,22 @@ spec:
   reporting-operator:
     spec:
       nodeSelector:
-        "node-role.kubernetes.io/infra": "true"
+        "node-role.kubernetes.io/infra": ""
```
Comment (lamek) [inline metering/configuring_metering/metering-common-config-options.adoc:85]: Let's add an annotation to the end of these lines and call out how the label works below the code block.

"node-role.kubernetes.io/infra": "" **<1>**

File: metering/configuring_metering/metering-common-config-options.adoc [hunk 2]
```diff
@@ -82,22 +82,22 @@ spec:
...
 
   presto:
     spec:
       coordinator:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": ""
```
Comment (lamek) [inline metering/configuring_metering/metering-common-config-options.adoc:91]: "node-role.kubernetes.io/infra": "" **<1>**

File: metering/configuring_metering/metering-common-config-options.adoc [hunk 3]
```diff
@@ -82,22 +82,22 @@ spec:
...
       worker:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": ""
```
Comment (lamek) [inline metering/configuring_metering/metering-common-config-options.adoc:94]: "node-role.kubernetes.io/infra": "" **<1>**

File: metering/configuring_metering/metering-common-config-options.adoc [hunk 4]
```diff
@@ -82,22 +82,22 @@ spec:
...
   hive:
     spec:
       metastore:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": ""
```
Comment (lamek) [inline metering/configuring_metering/metering-common-config-options.adoc:99]: "node-role.kubernetes.io/infra": "" **<1>**

File: metering/configuring_metering/metering-common-config-options.adoc [hunk 5]
```diff
@@ -82,22 +82,22 @@ spec:
...
       server:
         nodeSelector:
-          "node-role.kubernetes.io/infra": "true"
+          "node-role.kubernetes.io/infra": ""
```
Comment (lamek) [inline metering/configuring_metering/metering-common-config-options.adoc:102]: "node-role.kubernetes.io/infra": "" **<1>**

Comment (lamek) [reply metering/configuring_metering/metering-common-config-options.adoc:102]: And, below the code block after the closing dashes:

----
<1> Set this field to 'true' if you want to use a nodeSelector to specify a specific pod for your component to run on.

**Review by lamek — COMMENTED** [review]
One suggestion on clarifying what the label does.

File: metering/configuring_metering/metering-common-config-options.adoc (hunk 5 above)
Comment (timflannagan1) [reply metering/configuring_metering/metering-common-config-options.adoc:102]: Hmm, I'm not sure this still gets the message across that the node label provided in this configuration needs to exist on the set of nodes in order for the metering Pods to become scheduled.

Comment (timflannagan1) [reply metering/configuring_metering/metering-common-config-options.adoc:102]: It looks like the logging documentation essentially mirrors the kubernetes documentation for node selection: https://docs.openshift.com/container-platform/4.3/logging/cluster-logging-moving-nodes.html

Comment (lamek) [reply metering/configuring_metering/metering-common-config-options.adoc:102]: Logging note is good. I'll open a PR for this per your request.
https://github.com/openshift/openshift-docs/pull/21627

**Comment by timflannagan1** [comment]
Superseded by 21627

**Pull request closed without merging**